react-ui/src/App.js

```diff
@@ -1,23 +1,31 @@
 import logo from './logo.svg';
+import Button from 'react-bootstrap/Button';
```

Comment:
nit: _import { Button, Form, Row, Col } from "react-bootstrap";_

You can combine all the imports in one line.

Reply:
Done